downstreamadapter/eventcollector/helper.go

```diff
@@ -22,13 +22,18 @@ import (
 	"go.uber.org/zap"
 )
 
-func NewEventDynamicStream(collector *EventCollector) dynstream.DynamicStream[common.GID, common.DispatcherID, dispatcher.DispatcherEvent, *DispatcherStat, *EventsHandler] {
+func NewEventDynamicStream(collector *EventCollector) dynstream.DynamicStream[common.GID, common.DispatcherID, dispatcher.DispatcherEvent, *dispatcherStat, *EventsHandler] {
 	option := dynstream.NewOption()
 	option.BatchCount = 128
 	option.UseBuffer = true
 	// Enable memory control for dispatcher events dynamic stream.
-	log.Info("New EventDynamicStream, memory control is enabled")
 	option.EnableMemoryControl = true
+	if option.EnableMemoryControl {
```

Comment:
`EnableMemoryControl` is always true, so the log below is redundant?

Reply:
I gonna make it configurable.